swingbits/src/main/java/org/oxbow/swingbits/table/filter/TableRowFilterSupport.java

```diff
@@ -130,6 +132,14 @@ public TableRowFilterSupport onFilterChange(IFilterChangeListener listener) {
         return this;
     }
 
+    public ListCellRenderer getRenderer() {
+        return renderer;
+    }
+
+    public void setCheckListRenderer(ListCellRenderer renderer) {

```

Comment:
This method is part of fluent interface and not a setter, so to not confuse a user I would remove the word `set` and leave just `checkListRenderer`